### PR TITLE
Web 1488 add new digital brochure post type to replicate whitepapers

### PR DESCRIPTION
**Pull request by zubairali6**
Works similar to whitepapers feature except with added URL on confirmation page. 

To test:

- Go to wordpress and add new 'Digital Brochure' located in the admin menu on the left
- Fill in fields and publish
- Go to any page and add digital brochure (located under page resources metabox on page)
- Digital Brochure should appear on frontend page similar to whitepaper

### CODE REVIEW

File: public/wp-content/plugins/ccs-custom/library/custom-post-types.php [hunk 1]
```diff
@@ -249,4 +249,47 @@ function ccs_register_my_cpts() {
...
+        "rest_base" => "",
+        "has_archive" => true,
+        "show_in_menu" => true,
+        "exclude_from_search" => false,
+        "capability_type" => "whitepaper",
+        "map_meta_cap" => true,
+        "hierarchical" => false,
+        "rewrite" => array("slug" => "digital-brochure"),
+        "query_var" => true,
+        "menu_icon" => "dashicons-welcome-learn-more",
```
Comment (Chee-Ng) [inline public/wp-content/plugins/ccs-custom/library/custom-post-types.php:290]: Can you please change the icon to something else like:
    "menu_icon" => "dashicons-media-document",

File: public/wp-content/plugins/ccs-custom/library/custom-post-types.php [hunk 2]
```diff
@@ -249,4 +249,47 @@ function ccs_register_my_cpts() {
     );
 
     register_post_type( "event", $args) ;
+
+
+
+
+    // Digital Brochure(s) content type
```
Comment (Chee-Ng) [inline public/wp-content/plugins/ccs-custom/library/custom-post-types.php:256]: Also, can you move this section before the "event" so the ordering in CMS makes more sense

File: public/wp-content/plugins/ccs-custom/library/custom-post-types.php (hunk 1 above)
Comment (zubairali6) [reply public/wp-content/plugins/ccs-custom/library/custom-post-types.php:290]: ah yep that makes sense. Thanks.

File: public/wp-content/plugins/ccs-custom/library/custom-post-types.php (hunk 2 above)
Comment (zubairali6) [reply public/wp-content/plugins/ccs-custom/library/custom-post-types.php:256]: Sure, no problem.

**Review by Chee-Ng — APPROVED** [review]

**Pull request merged**